app/views/shared/_full_remission.html.slim

```diff
@@ -0,0 +1,2 @@
+#result.callout.callout-full
+  h3.bold =t('remissions.full').html_safe
```

Comment:
Instead of the line above, just have one line that works out what the remission result is and calls it. 

Just guessing here, but maybe:
```ruby
h3.bold =t("remissions.#{@overview.result}").html_safe
```
and the same repeated for the other two partials:

* _part_remission.html.slim
* _no_remission.html.slim

and going a step further and renaming _no_remission.html.slim to _none_remission.html.slim... Basically something that can derive the name of the partial based on the outcome of the remission. So that we avoid the 'translation'/mapping of the outcome to the partial.